.github/workflows/test-firedrake.yml

```diff
@@ -17,59 +17,87 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 60
     container:
-      image: firedrakeproject/firedrake:latest
+      image: firedrakeproject/firedrake-vanilla:latest
```

Comment:
Note that this points at an old image. It is now called `firedrake-vanilla-default:latest`.